CONTRIBUTING.md

```diff
@@ -232,8 +232,8 @@ Like any good open source project, we use Pull Requests (PRs) to track code chan
 3. Assigning reviews
     - Once a review has the `awaiting review` label, maintainers will review them as schedule
       permits. The maintainer who takes the issue should self-request a review.
-    - Any PR with the `size/large` label requires 2 review approvals from maintainers before it can
-      be merged. Those with `size/medium` or `size/small` are per the judgement of the maintainers.
+    - Any PR with the label `size/S` or larger requires 2 review approvals from maintainers before it can
```

Comment:
Just checking, didn't we want two LGTMs on all community-submitted PRs, and PRs submitted by core maintainers still only need one other LGTM? Or is that implicit here

Reply:
Yes, The next paragraph below clarifies this, though we can probably note that we mean "PRs from a community member" rather than "Any PR".